microcosm/annotations.py

```
@@ -0,0 +1,106 @@
+"""
+Configuration annotations.
+
+"""
+from distutils.util import strtobool
+
+
+def boolean(value):
+    """
+    Annotate a value as boolean.
+
+    """
+    if value == "":
```

`strtobool` fails hard on an empty string.